active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,591 @@
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            <muxc:TeachingTip.HeroContent>
+                <Image Source="Assets/Giraffe.png" />
+            </muxc:TeachingTip.HeroContent>
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipHeroContentSample.jpg" height="600"></a>
+
+### Add an icon
+
+An icon can be added to the side of the title and subtitle using the IconSource property.
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            IconSource="SaveIcon.png"
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipIconSample.jpg" height="600"></a>
+
+### Enable light-dismiss
+
+A teaching tip may be light-dismiss enabled so that it will dismiss when a user scrolls or interacts with other elements of the application.
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            IsLightDismissEnabled="True"
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipLightDismissSample.jpg" height="600"></a>
+
+### Preferred placement
+
+Preferred placement of a teaching tip can be set using the TeachingTipPlacementMode property. If the preferred placement would not leave enough room for the teaching tip to show, another placement mode will automatically be chosen.
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            PreferredPlacement="TopEdgeAlignedLeft"
```

Comment:
```suggestion
            Placement="TopEdgeAlignedLeft"
```
At least, I think that's what I'm seeing in the API...

Reply:
@YuliKl, thank you for the catch! As far as I am aware, this is a mistake in the API. "PreferredPlacement" was decided on to better alert developers upfront that their specified placement is subordinate to ensuring the tip will show at all. I would like to check with @StephenLPeters or @MikeHillberg to ensure this wasn't intentionally changed to better align with Flyout's placement API?

Reply:
I confirmed with @StephenLPeters that this is a bug and "Placement" should be "PreferredPlacement". Thanks again for the catch!